app/views/teachers/classrooms/students/edit_quiz_score.html.slim

```diff
@@ -0,0 +1,20 @@
+.row
+  .small-12.columns
+    .center.paper
+      .row
+        .small-12.columns
+          h2 = t ".heading", student_email: @student.email
+          .subtitle = t ".subtitle"
+
+          .spacer-small
+
+          = render "form_errors", obj: @student
+          = simple_form_for @student,
+            :url => "/students/#{@student.id}/edit_quiz_score/#{@quiz.id}" do |f|
```

Comment:
Nice find! You figured out that we can override the default url with the `url` key and you also figured out how to get the route you wanted in the form.

However, we don't like hardcoding URLs in our views. You made a named route for this action. It should look something like `student_quiz_score_path` or something like that. Check `rake routes` and then use that instead of a hardcoded URL.

The final line should look something like:

`url: student_quiz_score_path(@student, @quiz)`